aws/table_aws_dynamodb_table.go

```diff
@@ -191,6 +191,13 @@ func tableAwsDynamoDBTable(_ context.Context) *plugin.Table {
 				Hydrate:     getDynamoDBTable,
 				Transform:   transform.FromField("SSEDescription"),
 			},
+			{
+				Name:        "deletion_protection_enabled",
+				Description: "Indicates whether deletion protection is enabled (true) or disabled (false) on the table.",
+				Type:        proto.ColumnType_BOOL,
+				Hydrate:     getDynamoDBTable,
+				Transform:   transform.FromField("DeletionProtectionEnabled"),
```

Comment:
```suggestion
```

Reply:
Since the response field `DeletionProtectionEnabled` is already in the correct camel case format, no transformation is necessary here.